Automattic/wp-calypso
# Marketplace: Remove price toggle from search results

#### Changes proposed in this Pull Request

* Remove price toggle from plugins browser lists (search results & category pages but not individual plugin pages)

#### Testing instructions

* Should no longer see the price toggle on any browser pages

Before
![Screenshot 2022-05-23 at 15-07-09 Plugins — WordPress com](https://user-images.githubusercontent.com/811776/169747437-6d8625b4-9cc4-4cbd-8d3c-900cfa610310.png)

After
![Screenshot 2022-05-23 at 15-05-54 Plugins — WordPress com](https://user-images.githubusercontent.com/811776/169747447-48b208ae-7b0e-4b7a-834d-9ef4788c1a2a.png)

Fixes https://github.com/Automattic/wp-calypso/issues/63790
